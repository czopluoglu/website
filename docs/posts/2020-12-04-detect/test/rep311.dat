00000011001100101011
10111101110111010110
10000100110100010111
00011000001110111111
11101100101001111111
11100100001001100111
00010100011000100011
00000001001001101111
11011101111111001011
11010111011111101011
10110111110011111110
01100101111100111011
00001110011100001110
10100001111010101111
11011100100101110110
11100110110011011111
01110011001110111111
10111001111011101111
10001000101101111011
00101011011111100111
10001010100010010010
10100100111110110111
01111011000010000100
11001110000011111110
01101111110001100011
11000001010011100110
10111000111111000110
00011001011110011111
10010001111111111111
10101111000110101111
01110111011110111110
11111101101110110110
11111111011110100111
11001101000111011011
11000101010001001000
11000101001110100010
01111000111000111011
01111001110100111111
00001000101110000110
11101000011011100110
00100000110101011101
10110101111110110011
01110111111111111110
00100010000101101110
10000111100000100010
01000000101000001100
11010100001011101010
10110111110010010110
11110100011010001100
11100111101111111000
01000001101000100010
10000010010011001100
00100100001101111110
01001000100011100101
01001101111111111111
10011011111111111011
10000000001010001000
10001101011100111000
01001011101100010111
11101100001001101110
11111101011111100110
10001101000010100011
10101101001011101101
01110101111000000000
11000010101010111010
10011001000110100011
10010001110110000100
11101001111101111011
01010000001111101110
11010111001111001110
01001001011000110111
00001100001100111101
10001100000010101111
11101111010111101101
01111000100011001010
01000101011000000001
10000011001111011011
00011100111100100010
10111010111101111111
11011101101001100110
11110101111101101111
11010111011101110111
10101001001101111011
00100000110001100111
01110111101111111111
10010000001010110100
10001100001010101100
10001011111011000101
00110001001111111111
00101001111110101110
10110111001010010110
01000001011010111111
11011101010000001111
10000000000010101010
11111101001111111100
00110001011100001011
00110011111000101011
00111101101000110011
11001100110101000010
01000001001011110110
11001101011100010000
01101011111011100100
10000001010111100100
11000101011110010110
00111111111010001010
11100001101010001011
11001101000000000010
10001100001111000010
10111101100010111010
00010100000010000001
11101011111110110100
10000101011011111111
00000100001000000000
11110101111110101111
00011001010000000010
10100000001011010111
01011000101010110100
11111010111111000010
10001100011101010111
10110010111001000010
10010001010011000111
11010111111111101110
00001011000110111110
01010101111111110111
00000001111010000100
00001000011000001010
00000100011010101010
00001101001100010111
11100001111100111101
10011111011111111111
01100001011101010000
01001000111010111111
10100101111100000111
00011100011010110010
11110001101000000000
00001101000011100101
11011101011111111111
00010110001111101101
00100101001101110110
10110101111100000110
11101000100000001000
01001101010000001010
11101101111011101110
10101101111100011011
01001101011011111010
11100110011000101110
11101001101111011011
10111111011010101111
00000011000110101011
01000001011011100011
11110111101110110111
00000000011111110111
00000000001111101111
10011001111110101110
11011010100001111010
01011110101011111110
11111111110100111010
10111000101101111111
11111111001010100011
10011100000000101010
00000010101100100110
11100101110010000111
01110101111101100111
10010001011110101000
01111110111011111111
11101000101011110011
11100000111010101110
11100011001010100111
11111110111011100100
10000000110001000111
11001001101010110000
11101111111111111111
01100100010101101001
01010101010010100110
00101110011010100001
11011011111110001111
10101011000110111111
01000110011111111111
01000101001101111000
10011011001010110011
10000001110011100011
01111101111011010110
11110111010000100101
10001000011010000110
10100111111011101110
01001000001010000110
10000001110010000001
00101010111100101110
00100010101111101110
10000111011010101010
11110011111111010010
10100111111110110110
11100110000011111110
11011111111110111110
00010001011010001110
00110000001110100011
01000111011101101111
10110111110111110111
00010011101000101111
10010010001010101111
11111011111100111011
00011000000000101011
11011101011001001111
11111100110111101110
11111100000110010111
11101111001111110101
10111001111000011001
01111111101110110110
11010101101010111101
00001001011011100110
00100001110100101111
11110011111101001111
10111101110011001111
01001001000010101110
01001001111010101110
11100001001010101010
00001001101111001111
11000001001110111111
01100100111111000100
00110011010000011101
10101001010000101100
00011110000110111110
10110101111111100110
10001111000100111010
00011101110000101110
10101101000000010011
10110001101011101010
11000100010110001100
10100000001101100011
11011000111010001111
11100101111000001011
11111011101111111110
10001010001010111010
00110101111001100001
00100111101010101010
10111100110101101000
01100000110010100010
11111010100100010111
01001000000101100110
10101011100001101110
01011001101101110111
00000111111111011100
10000111111110110000
00011011111010110011
11011101011100101111
10101001011000001010
10101001011010100100
01011111111111101111
11100111110100000111
11101100110111100111
