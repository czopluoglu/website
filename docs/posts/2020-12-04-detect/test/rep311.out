-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep311.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.09
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7452
                   IDN index value:   0.6947
                           Ratio r:   0.6352

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    2    3    1    4    2    2    2
    5    5    4    1    4    5    4    4    5    5

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    6   14

   -----------CLUSTER  2-------------

    3    4    8    9   10

   -----------CLUSTER  3-------------

    5

   -----------CLUSTER  4-------------

    7   13   15   17   18

   -----------CLUSTER  5-------------

   11   12   16   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d-+++-+--+-+--------
   -d+++--+------++----
   ++d--++-+-------+---
   ++-d+--+---++-+-+--+
   ++-+d+-++-+---------
   --+-+d++++++---+--+-
   +-+--+d-+-+----+-+-+
   -+-+++-d+-----------
   --+-++++d-----++---+
   +----+---d--++---++-
   ----+++---d++++-+++-
   +--+-+----+d+-++----
   ---+-----+++d+++-+++
   ---------++-+d--+-++
   -+-+----+-+++-d-+--+
   -+---++-+--++--d++--
   --++------+--+++d+++
   ------+--++-+--++d-+
   -----+---++-++--+-d+
   ---+--+-+---+++-+++d
 
 No cross validation for this DETECT run
